docs/reuse/best-practices.txt

```diff
@@ -0,0 +1,36 @@
+**[How to run workloads with a machine charm](https://documentation.ubuntu.com/ops/latest/howto/run-workloads-with-a-charm-machines/)**
```

Comment:
The file seems formatted as markdown, but the extension is txt.
What am I missing?

Reply:
The starter-pack rule says to use that convention. It's at the end of [this section of the docs](https://canonical-starter-pack.readthedocs-hosted.com/latest/reference/rst-syntax-reference/#file-inclusion).

Reply:
Does reuse work when the file is markdown and not rst?